disnake/interactions/base.py

```diff
@@ -272,6 +274,16 @@ def permissions(self) -> Permissions:
             return Permissions(self._permissions)
         return Permissions.private_channel()
 
+    @property
+    def app_permissions(self) -> Permissions:
+        """:class:`Permissions`: The resolved permissions of ourselves in the channel, including overwrites.
```

Comment:
Would be great if the documentation clarified the difference between this and `inter.channel.permissions_for(inter.me)`, or lack thereof.

Reply:
https://github.com/DisnakeDev/disnake/pull/586/commits/876b4d67200f85a8d4e6a8713652d5cc12d579e3